packages/editor-ui/src/components/Node.vue

```diff
@@ -84,9 +91,21 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 
 			return workflowResultDataNode.length;
 		},
+		nodeOffsetPosition() {
+			return this.$store.getters.getNodeViewOffsetPosition;
+		},
 		isExecuting (): boolean {
 			return this.$store.getters.executingNode === this.data.name;
 		},
+		isSingleActiveTriggerNode (): boolean {
+			return this.$store.getters.getActiveWorkflowTriggerNodes.length === 1 && !this.data.disabled;
+		},
+		isTriggerNode (): boolean {
+			return this.nodeType !== null ? this.nodeType.group.includes('trigger') : false;
```

Comment:
nitpick - can simplify 
```suggestion
			return !!(this.nodeType && this.nodeType.group.includes('trigger'));
```
as with nitpicks always, it's up to you whether you want to make this change

Reply:
This is changed as well.